docs/docs/rest-api/public/api/v2/groups.raml

```diff
@@ -0,0 +1,376 @@
+###
+# Get the root group
+#
+get:
+  description:
+    Get the root group with all applications and all transitive child groups.
+  is: [ secured ]
+  responses:
+    200:
+      description: The root group with all transitive dependencies.
+      body:
+        application/json:
+          example: !include examples/group.json
+          schema: Group
+
+###
+# Update the root group
+#
+put:
+  description:
+    Change parameters of a deployed application group.
+    The new group parameters get applied.
+
+
+    * Changes to application parameters will result in a restart of this application.
+
+    * A new application added to the group is started.
+
+    * An existing application removed from the group gets stopped.
+
+
+    If there are no changes to the application definition, no restart is triggered.
+    During restart marathon keeps track, that the configured amount of minimal running instances are _always_ available.
+
+    This method alows 2 special modes for the update operation>
+
+    * Provide only the `version` field in the group definition. This will rollback the group to that given version
+
+    * Provide only the `scaleBy` field will scale all transitive applications by the given factor.
+
+    A deployment can run forever. This is the case, when the new application has a problem and does not become healthy.
+    In this case, human interaction is needed with 2 possible choices
+
+
+    * Rollback to an existing older version
+
+    * Update with a newer version of the group which does not have the problems of the old one.
+
+
+    If there is an upgrade process already in progress, a new update will be rejected unless the force flag is set.
+    With the force flag given, a running upgrade is terminated and a new one is started.
```

Comment:
We should describe somewhere, what happens with tasks created during a later on cancelled deployment :speak_no_evil: